events/events_network.c

```diff
@@ -136,6 +136,15 @@ events_network_register(int (*func)(void *), void * cookie, int s, int op)
 		goto err0;
 
 	/*
+	 * There is a new socket/operation pair:
+	 * Rebuild the fd list on the next call to events_network_select().
+	 */
+	if (fds != NULL) {
```

Comment:
Ah, I see. I'll think about this and see what I'll do about this, might take a bit (depending on my free time :smile:).